examples/kitchensink-ts/src/admin/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "lib": ["es2019", "es2020.promise", "es2020.bigint", "es2020.string", "DOM"],
+    "noImplicitAny": false,
+    "module": "es2020",
+    "target": "es5",
+    "jsx": "react",
+    "allowJs": true,
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "noEmit": false,
+    "incremental": true
+  },
+  "include": ["../plugins/**/admin/src/**/*", "./"],
```

Comment:
@Convly we will need to update the generators with the new config

Reply:
Alright :+1: 
I'll commit that